Off-Top-App/off-top-python
# OT-101 | Convert audio data/bytes to wav file in Python

# Task Completed 
- Created a file in python that is able to convert audio bytes to audio file in .wav format.

# Requirements for testing
- Python 3.7 
- Python libraries numpy and scipy

# How to test the branch
-  Open a terminal and cd into the Models folder inside python.
- Run `python byteToAudio.py` to run the byteToAudio file.
- If the terminal does not show up any errors then go inside the Models folder. There should be a .wav file called `format`. 
- We have successfully converted byte data from `test.txt` to audio file `format.wav`
